src/main/java/com/brightpath/learnify/controller/mapper/DtoMapper.java

```diff
@@ -124,6 +129,23 @@ public QuizSimpleResult asQuizSimpleResult(QuizResultUpdateDto quizResultUpdateD
         return new QuizSimpleResult(quizResultUpdateDto.getIncorrect(), quizResultUpdateDto.getCorrect());
     }
 
+    public CommentCreation asCommentCreation(CommentCreationDto commentCreationDto, ResourceType resourceType, UUID resourceId) {
+        return new CommentCreation(commentCreationDto.getOwnerId(),
+                resourceType,
+                resourceId,
+                commentCreationDto.getRating().shortValue(),
+                Optional.ofNullable(commentCreationDto.getTitle()),
```

Comment:
I think at least one of title and description should be required. ?

Reply:
rating is only reaquired thing